CMakeLists.txt

```diff
@@ -68,7 +65,16 @@ target_include_directories(
 
 if(SPDLOG_FMT_EXTERNAL)
     target_compile_definitions(spdlog INTERFACE SPDLOG_FMT_EXTERNAL)
-    target_link_libraries(spdlog INTERFACE fmt::fmt)
+
+    if(NOT TARGET fmt::fmt)
```

Comment:
why this addition?

Reply:
This puts everything relating to `SPDLOG_FMT_EXTERNAL` option into a single, neat block. See removed lines 58 - 60. Having it split up like that seemed like bad style to me.